mixer/pkg/config/crd/store.go

```diff
@@ -186,6 +187,31 @@ func (s *Store) extractCriticalKinds(r []string) []string {
 	return cks
 }
 
+func (s *Store) retryCreateCache(
+	d discovery.DiscoveryInterface,
+	lwBuilder listerWatcherBuilderInterface,
+	kinds []string) {
+	remaining := kinds
+	tick := time.Tick(s.retryInterval)
```

Comment:
The retry interval here should be much longer than init.
This loop will potentially run for ever.

Also can you add crds for cloudwatch and datadog? Those two always show up as remaining right?

Reply:
Added another variable for background retry interval, which by default is 10s.

Also Added crds for dogstatsd and cloudwatch.